plugins/node/instrumentation-fs/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@opentelemetry/instrumentation-fs",
+  "version": "0.27.0",
```

Comment:
Why starting at version 27?

Reply:
Out of a habit. We used to version-match packages. I guess it can now be `0.1.0`, which makes a lot more sense.